project/src/main/java/com/google/sps/data/PlacesFetcher.java

```diff
@@ -80,8 +68,7 @@ public PlacesFetcher() {
      * @throws ApiException
      */
     public List<Place> fetch() throws IOException, InterruptedException, ApiException {
```

Comment:
Continuing a conversation I cannot comment on anymore: 
why not accept the inputs as parameters in your library? and let the caller specify them? 
Otherwise you'll need another change to your library, although it's quite ready.

Reply:
It will be like that in M1 (as our [design](https://docs.google.com/document/d/1yPTb2Rh6MSTrDE9b9e1Q61ZCcH_cVNqBI8qs1VKNZEs/edit#) was planned. 
For now since in M0 we said it will all be hardcoded in the PlacesFetcher I don't see how inputs to fetch() are useful.